### ISSUE TITLE
IOS app crashes when call is timeout

### ISSUE THREAD

**Issue by kevinZartek** [hiennguyen92/flutter_callkit_incoming#438]
App crashes exactly when the incoming call is timed out ( call unattended ), crash is reported in testflight as PushKit: -[PKPushRegistry _terminateAppIfThereAreUnhandledVoIPPushes]

**Comment by hiennguyen92** [comment]
Please make sure call complete https://github.com/hiennguyen92/flutter_callkit_incoming/blob/590d0396d48012f504b9f28d9d113ed7318d0531/example/ios/Runner/AppDelegate.swift#L87

**Comment by kevinZartek** [comment]
Yes i have added this in the end of the method PushRegistry didReceiveIncomingPushWith.
